src/cloud-api-adaptor/podvm-mkosi/Makefile

```diff
@@ -82,25 +82,14 @@ ifeq ($(SE_BOOT),true)
 	touch resources/buildS390xImage
 	sudo mkosi --profile production.conf --image system
 	sudo -E ../hack/build-s390x-se-image.sh
-	@echo "Building docker image..."
-	sudo chown -R $(USER): build
-	docker buildx build \
-		-t $(PODVM_IMAGE) \
-		$(DOCKER_OPTS) \
-		-f ../podvm/Dockerfile.podvm.fedora .
 else ifeq ($(ARCH),s390x)
 	touch resources/buildS390xImage
 	sudo mkosi --profile production.conf --image system
 	sudo -E ../hack/build-s390x-image.sh
-	@echo "Building docker image..."
-	sudo chown -R $(USER): build
-	docker buildx build \
-		-t $(PODVM_IMAGE) \
-		$(DOCKER_OPTS) \
-		-f ../podvm/Dockerfile.podvm.fedora .
 else
 	touch resources/buildBootableImage
-	nix develop ..#podvm-mkosi --command mkosi --environment=VARIANT_ID=production
+	sudo -E env PATH=$(PATH) nix develop ..#podvm-mkosi --command mkosi --environment=VARIANT_ID=production
+	qemu-img convert -f raw -O qcow2 build/system.raw build/podvm-amd64.qcow2
```

Comment:
Could we have the qcow2's name close to the image's? For example, `podvm-generic-$(PODVM_DISTRO)-$(ARCH).qcow2`

Reply:
I think that's a good point, I don't think `generic` should actually be used anymore, if we are comfortable to drop vsphere support as everything use the "generic" image now, but we should reflect that in the podvm image name as well if everyone is happy.